cmd/agent/install_script.sh

```diff
@@ -85,8 +85,24 @@ Troubleshooting and basic usage information for the Agent are available at:
         read -t 60 -p  "Do you want to send a failure report to Datadog (including $logfile)? (y/[n]) " -r yn || on_read_error
         case $yn in
           [Yy]* )
-            read -p "Enter an email address so we can follow up: " -r email
-            report
+            cntr_max=3
+            cntr=0
+            email_regex='^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$'
```

Comment:
I think we should remove the `{2,4}` at the end and make it so that it accepts longer TLDs, you can see [here](https://en.wikipedia.org/wiki/List_of_Internet_top-level_domains#ICANN-era_generic_top-level_domains) that there are quite a few of them that are longer than 4 characters, and some of those are used in practice.

There are also other edge cases we are missing with this regex, e.g. `".john.smith."@[IPv6:2001:db8::1]` is valid and we reject it and `<something longer than 64 characters>@example.com` is invalid but we accept it, but I think it's fine since I would expect those to be extremely rare in practice.